common-pg/src/main/scala/com/socrata/pg/soql/SoQLAnalysisSqlizer.scala

```diff
@@ -355,7 +355,7 @@ trait SoQLAnalysisSqlizer {
         s" ${typ} ${lateralStr}$tableSql ON $joinCondition"
       }
     }
-    val (joinPhrase, setParamsJoin) = joins.foldLeft((Seq.empty[JoinSql], Seq.empty[SetParam])) { case ((sqls, setParamAcc), join) =>
+    val joinPhrasesAndParams = joins.foldLeft((Seq.empty[(JoinSql, Seq[SetParam])])) { case ((joinAcc), join) =>
```

Comment:
What I'm changing from here down to the `joinAcc :+ ...` line is this.  Before, we were generating a list of SQLs (one per join) and a list of parameters (all the parameters for all the joins in one list).  I'm changing it so that we keep the joins together with their parameters so we can move them around as a unit down below.